docs/assets/templates/notebooks/example-1-name.yml

```diff
@@ -0,0 +1,7 @@
+version: 1.2
+notebooks:
+  - name: simple
+    format: JUPYTER
+    language: PYTHON
```

Comment:
I think these values can be capitialized however the user wants, so I'd suggest we go with a more natural scheme ("Jupyter" and "Python"?)

Both the `format` and `language` fields are optional, and the values default to those we see here.  It's probably good to illustrate their use, but in a very simple example, they could be left out...